VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -227,14 +227,33 @@ def get_http_util(self):
         from HttpUtil import HttpUtil
         return HttpUtil(self.logger)
 
+	def get_wireserver_endpoint_uri(self):
+
+		wireserver_endpoint_file = '/var/lib/waagent/WireServerEndpoint'
+		wireserverIP = None
+
+		if os.path.exists(wireserver_endpoint_file):
+		    with open(wireserver_endpoint_file, 'r') as wip:
+			    wireserverIP = wip.read()
+			    self.logger.log("wireserverIP found in {0} = {1}".format(wireserver_endpoint_file, wireserverIP))
+
+		if wireserverIP == None:
+            self.logger.log("Using static wireServerIP from CommonVariables")
+		    wireserverIP = CommonVariables.wireserverIP
+
+		wireserver_endpoint = "http://" + wireserverIP + ":80/machine?comp=diskEncryptionData"
```

Comment:
maybe this format string (that ends in ":80/machine?comp=diskEncryptionData") can be a constant in Common.py as well